### PR TITLE
Use elementary Loki baseApp

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

**Comment by Alexander-Wilms** [comment]
Sorry, didn't see the PR.

bot, build

**Comment by bilelmoussaoui** [comment]
It seems that vte is not included. Will add that :+1:

**Comment by Alexander-Wilms** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
Is it wise to use the Loki BaseApp as we need also webkit2gtk here?

File: io.elementary.code.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
 	"app-id": "io.elementary.code",
-	"runtime": "org.gnome.Platform",
-	"runtime-version": "3.26",
-	"sdk": "org.gnome.Sdk",
+    "base": "io.elementary.Loki.BaseApp",
```
Comment (TingPing) [inline io.elementary.code.json]: Broken whitespace.

**Comment by TingPing** [comment]
> Is it wise to use the Loki BaseApp as we need also webkit2gtk here?

It should simply be failing then. You can use loki + gnome runtime.

**Comment by Alexander-Wilms** [comment]
bot, build

**Comment by Alexander-Wilms** [comment]
```
cd /run/build/code/plugins/pastebin && /app/bin/valac -C -b /run/build/code/plugins/pastebin -d /run/build/code/plugins/pastebin --pkg=libsoup-2.4 --pkg=granite --pkg=gtksourceview-3.0 --pkg=scratchcore --pkg=gee-0.8 --pkg=libpeas-1.0 --thread --vapidir=/run/build/code/vapi/ --vapidir=/run/build/code/src/ --target-glib=2.32 -g /run/build/code/plugins/pastebin/pastebin.vala /run/build/code/plugins/pastebin/pastebin_dialog.vala && touch /run/build/code/plugins/pastebin/pastebin_valac.stamp
/run/build/code/plugins/pastebin/pastebin_dialog.vala:27.35-27.61: error: The type name `Granite.Widgets.LightWindow' could not be found
    public class PasteBinDialog : Granite.Widgets.LightWindow {
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by bilelmoussaoui** [comment]
I'm not sure neither. It was using 0.5 before and the same version is shipped within io.elementary.Loki.BaseApp https://github.com/flathub/io.elementary.Loki.BaseApp/blob/master/io.elementary.Loki.BaseApp.json#L59

**Comment by bilelmoussaoui** [comment]
Shouldn't we force the Gtk/icon theme here too? to ensure that the application looks like the screenshots.

**Comment by bilelmoussaoui** [comment]
@Alexander-Wilms Any news on this?

**Comment by Alexander-Wilms** [comment]
I'd like to EOL this, since I'm more invested in the Qt eco system. Or would you want to take over?

**Comment by bilelmoussaoui** [comment]
@Alexander-Wilms I can take care of this :)

**Comment by Alexander-Wilms** [comment]
Thanks :) I've asked on IRC that the repo ownership be transferred.

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
The application fails because it uses features from Juno that are not present on the Loki's baseApp. Will create a Juno baseApp first and update the application here

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
@codygarver sorry to ping you here. Everything compiles & works perfectly without the plugins. When I include the plugins, the editorconfig one fails to build (the library is not detected) while it's correctly built & installed. Do you have any idea how to fix this?

**Comment by codygarver** [comment]
@decathorpe has experience with that library, maybe he has a hint.

**Comment by decathorpe** [comment]
You can look at the fedora packaging .specs for editorconfig and code, maybe it helps:

https://src.fedoraproject.org/rpms/editorconfig/blob/master/f/editorconfig.spec

https://src.fedoraproject.org/rpms/elementary-code/blob/master/f/elementary-code.spec

**Pull request closed without merging**